yarn-project/aztec.js/.gitignore

```diff
@@ -0,0 +1 @@
+src/account_contract/artifacts
```

Comment:
I'll double check, but given that we build-all-the-things in yarn-project, and in the bootstrap script, I would assume these should be present and not need to be committed.

As a separate point of note, I don't think these should even be here? The account contracts are defined by users of our system. Is there a reason we're baking these into aztec.js?

I could kind of see them being in the CLI, if we assume cli to be a layer that wraps aztec.js.

Reply:
Good point! We can probably move them elsewhere at this point. Not the CLI though: these contracts need to be available as a library, since they are used a lot in tests (to bootstrap accounts to interact with) or even in the sandbox itself.